tests/src/main/java/com/hazelcast/stabilizer/tests/map/IntIntMapTest.java

```diff
@@ -101,33 +91,23 @@ public void warmup() throws InterruptedException {
         }
     }
 
-    @Run
-    public void run() {
-        ThreadSpawner spawner = new ThreadSpawner(testContext.getTestId());
-        for (int k = 0; k < threadCount; k++) {
-            spawner.spawn(new Worker());
-        }
-        spawner.awaitCompletion();
-    }
-
-    @Performance
-    public long getOperationCount() {
-        return operations.get();
-    }
-
-    private class Worker implements Runnable {
-        private final Random random = new Random();
+    @RunWithBaseWorker
+    public BaseWorkerTask<Operation> createBaseWorker() {
```

Comment:
Please make of of an ordinary inner class. This thing is very hard to read. 'Never' use anonymous interclasses of more than a few lines.